numpy/_core/include/numpy/dtype_api.h

```diff
@@ -364,9 +364,12 @@ typedef int (PyArrayMethod_PromoterFunction)(PyObject *ufunc,
 #define NPY_DT_ensure_canonical 6
 #define NPY_DT_setitem 7
 #define NPY_DT_getitem 8
-#define NPY_DT_get_clear_loop 9
-#define NPY_DT_get_fill_zero_loop 10
-#define NPY_DT_finalize_descr 11
+#define NPY_DT_get_sort_function 9
+#define NPY_DT_get_argsort_function 10
+#define NPY_DT_sort_compare 11
+#define NPY_DT_get_clear_loop 12
+#define NPY_DT_get_fill_zero_loop 13
+#define NPY_DT_finalize_descr 14
```

Comment:
can you re-order these so the slots that were already in the struct keep their old values? I don't know offhand if changing this order is problematic but it seems more consistent to not change the old values even if it's fine.

Reply:
Yes, done!